.github/workflows/eval.yml

```diff
@@ -47,6 +47,9 @@ jobs:
         with:
           extra_nix_config: sandbox = true
 
+      - name: Ensure flake outputs on all systems still evaluate
+        run: nix --experimental-features 'nix-command flakes' flake check --all-systems --no-build
```

Comment:
can we move this into eval-aliases maybe? This step is on the critical path.

Reply:
I don't mind, but I intentionally put this on the critical path so that the flake keeps working, instead of breaking every few months :laughing:

Reply:
It will be still executed this way, but it makes the CI faster if we don't put it in the slow path.